docs/data-sources/aws_transit_gateway_attachment.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
The `docs/*` files included here were generated with `go generate`, which pulls in the examples and resource/field descriptions.